### PR TITLE
Cxc 315  implement website for project showcase

### PR DESCRIPTION
**Pull request by hollaa**
The whole website is created using only html and css
I was not able to recreate the exact same navbar and footer (using exactly the same css) as we use in our app but I tried to make it as similar as possible, please check what you think about it
Please review this task also on different devices (https://cxc-315.netlify.app/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,8 +1,231 @@
...
+          <div class="landing-text">
+            <h3>Words are good, pictures are better.</h3>
+            <h1>Create comics now!</h1>
+            <p>
+              Comics are a powerful way to bring stories to life and can even
+              support student engagement in learning complex scientific
+              subjects. With our user-friendly platform, anyone can become a
+              comic creator – no artistic skills required!
+            </p>
+            <button class="primaryButton" onclick="window.open('https://comixcraft.com/', '_blank')">Start comic crafting</button>
```
Comment (Vanessi-K) [inline index.html]: To comply with conventions and accessibility. Create this link over an a-tag

File: style.css [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+  padding: 0;
+  font-family: 'Nunito', sans-serif;
+  scroll-behavior: smooth;
+
+}
+
+.primaryButton {
+  background-color: #f460b7;
+  border: none;
+  border-radius: 8px;
```
Comment (Vanessi-K) [inline style.css:17]: Why does this button have a different border-radius than the button in the nav.

**Review by Vanessi-K — COMMENTED** [review]
I thik it would be good to have more spacing between the text and the gif. 
image Maybe also a slight border or shadow around the gif, so that it does not blend that much with the background.

I think we should use a FH logo without the text below, since at this sice you can not read it either way. 
image

In the footer instead of the "About comixcraft", which links to the same page we should have a link to our app. 
image

Maybe change the spacing/sizes so that there are four and three images in a row, instead of 5 and 2.
image


(This Review was done on Firefox on desktop -> Mobile will follow)

File: index.html [hunk 3]
```diff
@@ -1,8 +1,231 @@
- <!DOCTYPE html>
-<html>
-<body>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Comic editor</title>
```
Comment (Vanessi-K) [inline index.html]: I am not sure of this title

File: index.html [hunk 4]
```diff
@@ -1,8 +1,231 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200..1000;1,200..1000&display=swap"
+      rel="stylesheet"
+    />
```
Comment (Vanessi-K) [inline index.html]: Please host this yourself. This is GDPR incomplient.

File: index.html [hunk 5]
```diff
@@ -1,8 +1,231 @@
...
+            <p>
+              Comics are a powerful way to bring stories to life and can even
+              support student engagement in learning complex scientific
+              subjects. With our user-friendly platform, anyone can become a
+              comic creator – no artistic skills required!
+            </p>
+            <button class="primaryButton" onclick="window.open('https://comixcraft.com/', '_blank')">Start comic crafting</button>
+          </div>
+          <div class="landing-page-comixplain">
+            <img src="pictures/fh-logo.png" alt="fhstpLogo" />
```
Comment (Vanessi-K) [inline index.html]: Alt-text should be normal readable text (no camelcase)

File: index.html [hunk 6]
```diff
@@ -1,8 +1,231 @@
...
+          <img src="pictures/contributors/Anna.png" alt="" class="contributor-picture">
+          <div class="name">
+            <button class="linkedin-button" onclick="window.open('https://www.linkedin.com/in/anna-blasinger-3ba0a52bb/', '_blank')"  ><img src="pictures/LinkedinLogo.png" alt="linkedin link to the profile" ></button>
+            <p>Anna Blasinger</p>
+          </div>
+        </div>
+        <div class="one-contributor">
+          <img src="pictures/contributors/Florent.png" alt="" class="contributor-picture">
+          <div class="name">
+            <button class="linkedin-button" onclick="window.open('https://www.linkedin.com/in/florent-gabillon-2709b8194/', '_blank')"  ><img src="pictures/LinkedinLogo.png" alt="linkedin link to the profile" ></button>
```
Comment (Vanessi-K) [inline index.html]: The linkedin buttons should more be links than buttons since the link to a different page and not perform any actions.

**Review by Vanessi-K — CHANGES_REQUESTED** [review]

**Review by Vanessi-K — CHANGES_REQUESTED** [review]
I think on mobile the spacing between the FH sentence and the gif is a bit too little. It kinda seems as if the sentence about the FH is a heading for the gif.
image

File: index.html [hunk 7]
```diff
@@ -1,8 +1,225 @@
...
+    <footer class="footer mobile-container">
+      <div class="top_row mobile-container">
+          <button class="logo-button" onclick="window.open('https://comixcraft.com/', '_blank')">
+            <img class="logo" src="/pictures/comixcraftLogo.svg" />
+          </button>
+              
+          <div class="link_container">
+              <p class="link_text" onclick="window.open('https://comixcraft.com/impresum', '_blank')" class="link_text">Impressum</p>
+              <p class="link_text" onclick="window.open('https://comixcraft.com/', '_blank')" class="link_text">comixcraft</p>
+              <p class="link_text" onclick="window.open('https://fhstp.github.io/comixplain/', '_blank')" >About Comixplain</p>
```
Comment (Vanessi-K) [inline index.html]: These also should be a-tags. 
Also the link to the impressum is wrong -> you are missing a second s

File: style.css (hunk 2 above)
Comment (Vanessi-K) [reply style.css:17]: I don't see that this is changed

File: index.html [hunk 8]
```diff
@@ -1,8 +1,225 @@
...
+        </div>
+      </div>
+    </div>
+    <div class="github-container">
+      <img src="pictures/github.png" alt="github logo">
+      <p>Comixcraft is an open-source project and the repository is accessible on our <a href="https://github.com/comixcraft/Editor" target="_blank" rel="noopener noreferrer"> Github account</a>.</p>
+    </div>
+    <footer class="footer mobile-container">
+      <div class="top_row mobile-container">
+          <button class="logo-button" onclick="window.open('https://comixcraft.com/', '_blank')">
```
Comment (Vanessi-K) [inline index.html]: Also this is not a button but rather just a link

File: fonts/Nunito/README.txt [hunk 9]
```diff
@@ -0,0 +1,79 @@
+Nunito Variable Font
+====================
+
+This download contains Nunito as both variable fonts and static fonts.
```
Comment (Vanessi-K) [inline fonts/Nunito/README.txt]: I don't think that this file is necessary

**Review by Vanessi-K — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (hollaa) [reply style.css:17]: Ive changed it now but in the other button so it matches the buttons in the editor

**Comment by Vanessi-K** [comment]
I just saw, the headings are not in one line: 
image

**Comment by Vanessi-K** [comment]
Everything fine now

**Review by Vanessi-K — APPROVED** [review]

**Pull request merged**